src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  status: 'Under construction',
+  categories: [],
+};
```

Comment:
- Kindly make sure your initial state is empty at the beginning. It must be changed only when the event is triggered to update the application status. so delete the status 👍👌

Reply:
Thanks for this information, I will change the status to empty